File: drive_java/src/main/java/com/github/ximik3/gradle/DrivePluginExtension.java
package com.github.ximik3.gradle;

import java.util.HashMap;

public class DrivePluginExtension {
    int type = 0;
    /**
     * @see PluginType
     */
    int apkUpgradeType = 0;
    /**
     * @see ApkUpgradeType
     */
    String version = "";
    String account;
    String certificateFile;
    String keyStorePassword;
    HashMap<String, String> folderIds;
}

File: drive_kotlin/src/main/kotlin/com/github/ximik3/gradle/DrivePluginExtension.kt
package com.github.ximik3.gradle

import java.io.File

open class DrivePluginExtension {
    /** @see PluginType */
    var type = 0

    /** @see ApkUpgradeType */
    var apkUpgradeType = 0
    var version: String = ""
    var folderId: String? = null
    var serviceAccountEmail: String? = null
    var pk12File: File? = null
    var certificateFile: String? = null
    var keyStorePassword: String? = null
}